dbld/images/packages/requirements.txt

```diff
@@ -0,0 +1,6 @@
+#make_check
+hy                      [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+nose                    [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+pep8                    [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+ply                     [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+pylint                  [centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
```

Comment:
in this way, this is not in pip format anymore, so I would at the very least use a different name for the file then.